ui/Screen/Onboarding.svelte

```diff
@@ -46,12 +45,17 @@
       // Retry until the API is up
       identity = await withRetry(() => createIdentity({ handle }), 200);
       state = State.SuccessView;
-    } catch (error) {
+    } catch (err) {
       animateBackward();
       state = State.EnterName;
-      notification.error(
-        `Could not create identity: ${urn.shorten(error.message)}`
-      );
+      error.show({
+        code: "create-identity-failed",
```

Comment:
Are we eventually going to give these actual codes?

Reply:
Discussing in https://github.com/radicle-dev/radicle-upstream/pull/1221#discussion_r521412590